src/LumenGenerator/Console/GeneratorCommand.php

```diff
@@ -274,16 +274,25 @@ protected function getArguments()
     }
 
     /**
-     * Get the model's namespace baed on the app/Models folder.
+     * Qualify the given model class base name.
      *
+     * @param  string  $model
      * @return string
      */
-    protected function getModelNamespace()
+    protected function qualifyModel(string $model)
```

Comment:
Note that the `string` type hint require PHP 7.
With `illuminate/support:5.5` as minimum requirement, this wouldn't be a problem.